js/stores/appStore.js

```diff
@@ -503,6 +503,26 @@ const handleAppAction = (action) => {
     case AppConstants.APP_SET_DICTIONARY:
       appState = appState.setIn(['dictionary', 'locale'], action.locale)
       break
+    case AppConstants.APP_CLEAR_DATA:
+      if (action.clearDataDetail.get('browserHistory')) {
+        handleAppAction({actionType: AppConstants.APP_CLEAR_SITES_WITHOUT_TAGS})
```

Comment:
the browser history is still available in the `history` property on the frame. after i clear browser history, my session-store-1-dev file has entries like:
```
"frames":[{"canGoBack":false,"canGoForward":false,"computedThemeColor":"rgb(51, 51, 51)","endLoadTime":1470784542553,"findDetail":{"caseSensitivity":false,"searchString":""},"fingerprintingProtection":{},"history":["http://samy.pl/evercookie/","https://www.google.com/#q=abcdyahoo","https://www.google.com/#q=abcdyahoo","https://www.bing.com/"], ...
```